samples/client/petstore/typescript-axios/builds/composed-schemas/api.ts

```diff
@@ -221,7 +221,7 @@ export type PetsFilteredPatchRequestPetTypeEnum = typeof PetsFilteredPatchReques
  * @type PetsPatchRequest
  * @export
  */
-export type PetsPatchRequest = Cat | Dog | any;
+export type PetsPatchRequest = Cat | Dog;
```

Comment:
FYI. This PR results in changes in typescript-axios output, which can be reverted by setting the openapi-normalizer rule accordingly.

@TiFu (2017/07) @taxpon (2017/07) @sebastianhaas (2017/07) @kenisteward (2017/07) @Vrolijkx (2017/09) @macjohnny (2018/01) @topce (2018/10) @akehir (2019/07) @petejohansonxo (2019/11) @amakhrov (2020/02) @davidgamero (2022/03) @mkusaka (2022/04)